### PR TITLE
Conor

### PR DESCRIPTION
**Pull request by cohalloran1**

### CODE REVIEW

File: notebooks/generate_bracket.py [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+# Sample call
+
+# b = build_bracket(
+#     output_path="2021_bracket.png",
+#     teamsPath="../../data/MTeams.csv",
+#     seedsPath="../../data/MNCAATourneySeeds.csv",
+#     slotsPath="../../data/MNCAATourneySlots.csv",
+#     submissionPath="ncaa-march-madness-submission.csv",
+#     year=2021
+# )
```
Comment (cohalloran1) [inline notebooks/generate_bracket.py]: The bracketed stuff was expecting slightly different naming conventions for some columns, I don't think it's been updated. I just kinda borrowed the whole thing, made some edits, and confirmed it was still working.

File: notebooks/2021_final_submission/ncaa-march-madness-2021.ipynb [hunk 2]
```diff
@@ -0,0 +1,1273 @@
...
+    "from .generate_bracket import build_bracket\n",
+    "\n",
+    "b = build_bracket(\n",
+    "    output_path=\"2021_bracket.png\",\n",
+    "    teamsPath=\"../../data/MTeams.csv\",\n",
+    "    seedsPath=\"../../data/MNCAATourneySeeds.csv\",\n",
+    "    slotsPath=\"../../data/MNCAATourneySlots.csv\",\n",
+    "    submissionPath=\"ncaa-march-madness-submission.csv\",\n",
+    "    year=2021\n",
+    ")"
```
Comment (cohalloran1) [inline notebooks/2021_final_submission/ncaa-march-madness-2021.ipynb:1262]: Sorry this is so gross, but the only issue I've had is being able to access the build_bracket function from generate_bracket.py. I figured making it common would be nice for trying out different models, but I keep getting modulenotfound errors. Maybe you can tell what I'm doing wrong.

**Comment by cohalloran1** [comment]
@akhoch34 there's a couple comments explaining what I've gotten done. I was able to get my local .venv set up to handle the 2021_final_submission notebook we made, and spit out a bracket image. Next, I want it to do that by importing build_bracket from generate_bracket.py; to get it currently, I have to run the build_bracket call from generate_bracket.py directly. 

For 2021_top_performer (not in this PR because I didn't mess with it), that's the notebook that got 9th place or something last year. It was going fine until I tried to use lightgbm. I think that's the heavy lifting for the model, some sort of gradient boosting package. Maybe we could get away with sklearn? Unsure...but it looks like Macs have issues installing it. You can use homebrew, but I haven't been able to figure out how to use homebrew where it saves to my .venv. I wanna keep stuff local for git changes, but we might have to push that one up to Colab or something and run it there. Curious if you have any thoughts

File: notebooks/2021_final_submission/ncaa-march-madness-2021.ipynb (hunk 2 above)
Comment (cohalloran1) [reply notebooks/2021_final_submission/ncaa-march-madness-2021.ipynb:1262]: Jk I got it

**Comment by cohalloran1** [comment]
Fixed the build bracket module stuff, gonna work on lightgbm again and see what I can do
